styles.css

```diff
@@ -679,4 +764,66 @@ header .logo {
     animation-duration: 2s;
     animation-timing-function: ease-in-out;
   }
+
+  .modal {
+    margin: 60px 160px 60px 160px;
+    padding: 20px;
+    overflow: hidden;
+  }
+
+  .modal-top .item-title {
+    margin-top: -12px;
+    font-size: 24px;
+  }
+
+  .modal-top .item-history {
+    margin-top: -20px;
+  }
```

Comment:
- [ ] Kindly add some space between items here, to match the Figma design.

Your design
![image](https://user-images.githubusercontent.com/35535130/161875310-6287b8a6-e181-41d2-b259-c2eb059ea8f3.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/161875384-6d007072-cc24-4b07-b68f-6dda191def06.png)